package-lock.json

```diff
@@ -7530,9 +7530,9 @@
       }
     },
     "node_modules/mermaid": {
-      "version": "10.5.0",
-      "resolved": "https://registry.npmjs.org/mermaid/-/mermaid-10.5.0.tgz",
-      "integrity": "sha512-9l0o1uUod78D3/FVYPGSsgV+Z0tSnzLBDiC9rVzvelPxuO80HbN1oDr9ofpPETQy9XpypPQa26fr09VzEPfvWA==",
+      "version": "10.6.1",
+      "resolved": "https://registry.npmmirror.com/mermaid/-/mermaid-10.6.1.tgz",
```

Comment:
@wxiaoguang Don't think it causes any issues but we should avoid writing non-official npm mirrors in the lockfile.

Reply:
Agree, maybe it's because at that time I was using npmmirror. Will pay more attention

Reply:
If it becomes a problem, a simple find-replace should be able to fix it. I don't think the npm client actually looks at it, but other clients might.

Reply:
https://github.com/go-gitea/gitea/pull/30719 adds a linter for it.